test/specs/scenarios/sharedFlows.js

```diff
@@ -0,0 +1,15 @@
+import { localhostUrl } from '../../utils/config'
+
+export const goToPassportUploadScreen = async (driver, screens, parameter='') => {
```

Comment:
I wonder whether it would make sense to move these `const`s to `Common.js` class. There are no tests in this file.

Reply:
Indeed. I will move them there

Reply:
I think these shared flows require a better design. It can be handled in a different PR as this one is getting too big for its estimation.